tests/unit_tests/testaddon.cpp

```diff
@@ -690,12 +699,34 @@ void TestAddon::message_load_status_data() {
 }
 
 void TestAddon::message_load_status() {
+  Localizer l;
```

Comment:
I don't think this is used anywhere

Reply:
This is a pattern in out tests. The constructor has side effects taht are required for the test to run correctly.